### PR TITLE
Created Text utilities and lists

### PR DESCRIPTION
**Pull request by aryanandarnav13**

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
Good to merge

File: app.css [hunk 1]
```diff
@@ -6,12 +6,14 @@
...
 @import url("css/input.css");
+@import url("css/text.css");
+@import url("css/list.css");
 
 
 :root{
     --primary-color: rgb(81, 97, 116);
     --secondary-color:white;
-    --cover-color:rgb(224, 222, 222);
+    --cover-color:#c7dde6;
```
Comment (yash9263) [inline app.css:16]: use only one format for colors

File: app.css [hunk 2]
```diff
@@ -6,12 +6,14 @@
 @import url("css/card.css");
 @import url("css/image.css");
 @import url("css/input.css");
+@import url("css/text.css");
+@import url("css/list.css");
 
 
 :root{
     --primary-color: rgb(81, 97, 116);
     --secondary-color:white;
```
Comment (yash9263) [inline app.css:15]: don't use word values

File: css/text.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.gt1{
+background-color:rgb(16,16,16) ;
+color: white;
```
Comment (yash9263) [inline css/text.css:3]: fix indentation

**Review by yash9263 — APPROVED** [review]

**Comment by yash9263** [comment]
Other than the comments made everything looks good

File: app.css (hunk 1 above)
Comment (aryanandarnav13) [reply app.css:16]: Yaah at last ,I'm gonna correct all the color issues collectively.

File: app.css (hunk 2 above)
Comment (aryanandarnav13) [reply app.css:15]: Noted!!

**Comment by aryanandarnav13** [comment]
> Other than the comments made everything looks good

Thankyou so much for the review.I have listed all the correction.

**Pull request merged**